cluster/senza-definition.yaml

```diff
@@ -176,15 +180,17 @@ Resources:
                 -
                   - arn:aws:iam::{{ AccountInfo.AccountID }}:role/
                   -
-                    Ref: WorkerIAMRole
+                    Ref: MasterIAMRole
```

Comment:
Is this done by intention or a bug/leftover?
I did not read something like this in the change description.

Reply:
The autoscaler now runs on the master nodes instead of the workers. This should, in theory, allow us to downscale to 0 nodes using the normal downscaler.